hack/verify-kep-structure.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Check that the required section headers (from the kep template) are
+# present in the kep.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+TEMPLATE=$(dirname $0)/../keps/NNNN-kep-template/README.md
+
+MODIFIED_KEPS=$(git log --name-only --pretty= "${PULL_BASE_SHA:-origin/master}.." -- |
```

Comment:
- I'd make the default branch name (`master`) configurable via a variable, or work it out somehow.
- Is it safe to assume that there is a remote called `origin`?